pycbc/io/sqlite.py

```diff
@@ -54,16 +54,16 @@ def restrict_rows(self, conditions=None, limit=None):
                     self.restriction += " AND " # all subsequent conditions need an AND
```

Comment:
Yes I do not see why it would still be useful, but I wanted the change to be as minimal as possible. I am happy to remove it, let's hear from Collin.